aperturedb/Connector.py

```diff
@@ -113,13 +115,51 @@ def _authenticate(self, user, password="", token=""):
             raise Exception(
                 "Either password or token must be specified for authentication")
 
+
         response, _ = self.query(query)
 
         self.session = response[0]["Authenticate"]
         if self.session["status"] != 0:
             raise Exception(self.session["info"])
 
+        self.authenticate_timestamp = time.time()
         self.session_token = self.session["session_token"]
+        self.refresh_token = self.session["refresh_token"]
```

Comment:
this does not seem to be used in the code, as it's always looked up from self.session .